left_column.md

```diff
@@ -112,40 +112,43 @@ The schedule is subject to change.
   (TBA)
 - 11:30 - 12:00
   [Collaborative distributed version control 1/2](https://coderefinery.github.io/git-collaborative/)
-  (TBA)
+  (Radovan, Bjørn)
```

Comment:
I think this will be basically merging branches, conflicts, and history inspection. I would like to take the history inspection part. I think Bjørn wanted to talk about forks which would happen on May 27.